Source/Scene/Expression.js

```diff
@@ -45,6 +45,13 @@ define([
         }
     };
 
+    var binaryFunctions = {
+        atan2 : Math.atan,
```

Comment:
This should still be `Math.atan2`. Only the glsl side needs to change to `atan`.